R/tune_results.R

```diff
@@ -15,6 +15,24 @@ print.tune_results <- function(x, ...) {
                                 sep = "\n"))
     rlang::warn(print_notes)
   }
-  class(x) <- class(x)[!(class(x) %in% c("tune_results"))]
-  print(x, ...)
+
+  if (inherits(x, "resample_results")) {
+    cat("# Resampling results\n")
+  } else {
+    cat("# Tuning results\n")
+  }
+
+  att <- attributes(x)
+  if (any(names(att) == "rset_info")) {
+    cat("#", att$rset_info$label, "\n")
+  } else {
+    if (inherits(x, "rset")) {
```

Comment:
It should never inherit from rset anymore right? Is this to be backwards compatible? If so, I would extract it into a helper called `print_tune_results_compat()` or something to make it super clear

Reply:
The print method works for older objects and I think that is a good idea. The helpers should work too. 

`print_tune_results_compat()` is fine with me.